python/ray/scripts/scripts.py

```diff
@@ -615,6 +624,10 @@ def start(
             "All the worker nodes will use the same temp_dir as a head node. "
         )
         temp_dir = None
+    if start_gcs is True and not head:
+        cli_logger.error(
+            "{} and can be be used with {}.", cf.bold("--start-gcs"), cf.bold("--head")
```

Comment:
If I translate this, it is -> --start-gcs and can be be used with --head.

Maybe we should just write

```
--gcs-only and --head cannot be used together. 
```

Also, why don't we hard fail in this case?